src/displayBackend/drm/Dumb.hpp

```diff
@@ -290,9 +290,9 @@ class DumbZCopyBack : public DumbBase
 
 	void getBufDrmFd();
 
-	void getGrantRefs(domid_t domId, DisplayItf::GrantRefs& refs);
+	void getGrantRefs(domid_t domId, GrantRefs& refs);
 
-	void init(uint32_t bpp, domid_t domId, DisplayItf::GrantRefs& refs);
+	void init(uint32_t bpp, domid_t domId, GrantRefs& refs);
```

Comment:
maybe better to have a factory and return the ...ptr  at GrantRefs ?

Reply:
This is a pure code move, no changes to behavior. Please file a bug